src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/conditions/ExclusionZoneService.kt

```diff
@@ -126,6 +133,14 @@ class ExclusionZoneService(
     additionalConditionRepository.saveAndFlush(updatedAdditionalCondition)
   }
 
+  private fun removeExistingDbRows(additionalCondition: AdditionalCondition) {
```

Comment:
meh.. would rename to remove reference to DB, maybe `removeExistingUploadDetail`

(we're supposed to be agnostic at this level)

Reply:
is this something I can put forward in a future change?